man/fi_mr.3.md

```diff
@@ -281,6 +247,97 @@ memory region is based on the bitwise OR of the following flags.
 : Generates an event whenever a remote RMA write or atomic operation
   modify the memory region.
 
+# MEMORY REGION ATTRIBUTES
+
+Memory regions are created using the following attributes.  The struct
+fi_mr_attr is passed into fi_mr_regattr, but individual fields also
+apply to other memory registration calls, with the fields passed directly
+into calls as function parameters.
+
+```c
+struct fi_mr_attr {
+	const struct iovec *mr_iov;
+	size_t             iov_count;
+	uint64_t           access;
+	uint64_t           requested_key;
+	void               *context;
+	size_t             auth_keylen;
+	uint8_t            *auth_key;
+};
```

Comment:
Since the regattr struct has changed, do you intend to make a fabric_1_0_* version of this function for backward compatibility?